src/app/compose/compose.component.ts

```diff
@@ -186,6 +187,7 @@ export class ComposeComponent implements AfterViewInit, OnDestroy, OnInit {
 
     public uploadFiles(files: File[]) {
         this.uploadprogress = 0;
+        this.uploadingFiles = files;
```

Comment:
Just for the sake of "cleanliness", I'd `null`-ify it once the uploads are finished (like we do with `uploadProgress`)

Reply:
Yes of course, everything I changed. I just dont know how to change the message of the commit.

Reply:
@lauramurinova You can use `git commit --amend` to change the message. If you want to squash multiple commits into one you can use `git rebase -i master` ( and you'll be able to edit the commits since master )

Reply:
@petersalomonsen Thank you very much, unfortunately I already closed this one and created a new PR, where I applied all the changes. But for the future, I will now know.